web/views/note_view.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  def render_markdown(markdown) do
+    import Earmark
+    Earmark.to_html(markdown) |> raw
```

Comment:
I'm not understanding what `put_change` and `fetch_field` are going to do for you. The Markdown is generating HTML that needs to be rendered raw (if you escape it, you'll see the HTML tags on the page, which you don't want). You just want to strip out anything malicious. Check out https://github.com/rrrene/html_sanitize_ex.